spec.md

```diff
@@ -380,13 +380,43 @@ Example error response (invalid `id` provided):
 
 ## Collections
 
+A collection is a namespace for Resources, and a unit of configuration, within
+a space.
+
+In other storage systems, a collection is also known as: Directory, Folder,
+RDBMS Table, Document Collection, Graph, WebAPI FileList, Bucket, Solid
+Container, EDV Vault, DWN Collection, and so on.
+
+The use of collections is optional -- when a Space is created, the default `/`
+collection is automatically created within that Space.
+
 ### Collection Data Model
 
-#### Collection JSON Representation (ActivityPub profile)
+Collection properties:
 
-### Collection Operations
+* `id`
+* `name` - human readable shortname, url slug
+* `type` - A sorted array of strings, MUST include the type `Collection`.
+* `contents`
+* (tbd) a link to the **system meta** auxiliary object
+* (tbd) one or more links to controller-modifiable **custom meta** auxiliary
+  objects
+
+#### Collection JSON Representation (Activity Streams 2 profile)
+
+Example empty collection, in AS2 format:
 
-#### Collection HTTP API
+```js
+{
+  // id?
+  // name?
```

Comment:
idk if you meant to leave these in?

I think it would be ideal for this to be valid JSON, e.g. by removing these two lines.

tbh I am also down to not even define these two properties on here right now.

Reply:
I think I meant to leave them in as questions to you - like, I'm fairly sure collections have IDs, right? And I think you were on board with them having a name? Like "credentials" etc?

Reply:
I removed the placeholders.